clients/java/signalr/src/main/java/ConsoleLogger.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public class ConsoleLogger implements Logger {
+    private LogLevel logLevel;
+    public ConsoleLogger(LogLevel logLevel) {
+            this.logLevel = logLevel;
+    }
+
+    @Override
+    public void log(LogLevel logLevel, String message) {
+        if(logLevel.value >= this.logLevel.value){
+            switch (logLevel) {
+                case Debug:
```

Comment:
Just  copied the log levels from the TS client
https://github.com/aspnet/SignalR/blob/32d380a05801bda255bbf9d758298b8285e52ea5/clients/ts/signalr/src/ILogger.ts#L9-L24

Reply:
We can probably just consolidate these to Info and Error

Reply:
We wouldn't want to consolidate them, then we'd lose the ability to have more granular messages.